packages/11ty/_plugins/transforms/outputs/epub/transform.js

```diff
@@ -0,0 +1,70 @@
+const filterOutputs = require('../filter.js')
+const getIIIFConfig = require('../../../figures/iiif/config')
+const jsdom = require('jsdom')
+const layout = require('./layout')
+const path = require('path')
+const write = require('./write')
+
+const { JSDOM } = jsdom
+
+/**
+ * Content transforms for EPUB output
+ */
+module.exports = function(eleventyConfig, collections, content) {
+  const { output: iiifOutputDir } = getIIIFConfig(eleventyConfig).dirs
+  const slugify = eleventyConfig.getFilter('slugify')
+  const { imageDir } = eleventyConfig.globalData.config.params
+  const { language } = eleventyConfig.globalData.publication
+
+  /**
+   * Transform image asset paths to be relative to epub directory
+   *
+   * @param      {HTMLElement}  element
+   */
+  const transformImagePaths = (element) => {
+    const nodes = element.querySelectorAll('img')
+    nodes.forEach((img) => {
+      const url = img.getAttribute('src')
+      if (!url) return
+      const replace = `^\/(${iiifOutputDir}|${imageDir})`
+      const regex = new RegExp(replace, 'g')
+      img.setAttribute('src', url.replace(regex, '../$1'))
+    })
+    return element
+  }
```

Comment:
Not totally sure if this is the way to go, or if we should move assets into `/epub` 🤷‍♀️

Reply:
After our conversation with Fred, I started working on a branch that copies assets into `epub` as we come across them during the transform and add the paths to the list of assets in the manifest, it will just need to be updated to match the expected directory structure when we know what that should be